src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/model/PrintResponseBuilder.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dluhc.printapi.testsupport.testdata.model
+
+import uk.gov.dluhc.printapi.printprovider.models.BatchResponse
+import uk.gov.dluhc.printapi.printprovider.models.PrintResponse
+import uk.gov.dluhc.printapi.printprovider.models.PrintResponses
+import uk.gov.dluhc.printapi.testsupport.testdata.DataFaker
+import java.util.Date
+import java.util.UUID
+import kotlin.random.Random
+
+fun buildPrintResponses(
+    batchResponses: List<BatchResponse> = listOf(buildBatchResponse()),
+    printResponses: List<PrintResponse> = listOf(buildPrintResponse()),
+): PrintResponses =
+    PrintResponses()
+        .withBatchResponses(batchResponses)
+        .withPrintResponses(printResponses)
+
+fun buildBatchResponse(
+    batchId: String = Random.nextInt().toString(),
+    message: String = DataFaker.faker.famousLastWords().lastWords(),
+    status: BatchResponse.Status = BatchResponse.Status.SUCCESS,
+    timestamp: Date = Date(),
+) = BatchResponse()
+    .withBatchId(batchId)
+    .withMessage(message)
+    .withStatus(status)
+    .withTimestamp(timestamp)
+
+fun buildPrintResponse(
+    requestId: UUID = UUID.randomUUID(),
```

Comment:
I dont think requestId should be a UUID either .... 🤔 
It's meant to be a 24 char hexstring, hence we use mongo's ObjectId

Something's not right (in the yamlschema I suspect)